File: JavaScript/8js 1/distancePoints.js
/*
|--------------------------------------------------------------------------
| Geometry Basics: Distance between points in 2D
|--------------------------------------------------------------------------
|
*/

function distanceBetweenPoints(a, b) {
   return Math.sqrt((a.x - b.x) ** 2 + (a.y - b.y) ** 2);
}


const distanceBetweenPoints = ({ x: x1, y: y1 }, { x: x2, y: y2 }) => Math.hypot(x1 - x2, y1 - y2);


function distanceBetweenPoints(a, b) {
   return Math.hypot(a.x - b.x, a.y - b.y);
}

File: Python/8py 1/distance_points.py
"""
|--------------------------------------------------------------------------
| Geometry Basics: Distance between points in 2D
|--------------------------------------------------------------------------
|
"""

from math import sqrt, hypot, dist
def distance_between_points(a, b):
   return sqrt((a.x - b.x) ** 2 + (a.y - b.y) ** 2)


def distance_between_points(a, b):
   return hypot(a.x - b.x, a.y - b.y)


def distance_between_points(a, b):
   return dist((a.x, a.y), (b.x, b.y))